server/src/main/java/org/tctalent/server/model/db/Candidate.java

```diff
@@ -705,6 +707,12 @@ public class Candidate extends AbstractAuditableDomainObject<Long> {
     @Nullable
     private String mediaWillingness;
 
+    @NotNull
+    private Boolean emailConsentRegistration;
```

Comment:
Just thinking more about the naming again - would it be better to go with the more inclusie "contactConsentRegistration". Given that candidates could be contacted by other means than email.

Also instead of emailConsentPartners - go with contactConsentPartners?